zerver/webhooks/azuredevops/doc.md

```diff
@@ -28,10 +28,6 @@ Get Azure DevOps notifications in Zulip!
 
 {!event-filtering-additional-feature.md!}
 
-### Configuration options
```

Comment:
Approaches considered for updating the docs:
- Edit them as done in #33737, removing the macro. Since all of these except the GitHub doc use the exact same lines, considered adding a new macro with that 1 line appended to the generate-url macro instruction.
- But decided to leave it out, because "You'll be able to configure which branches you'll receive notifications from" is less of an instruction than something like "Uncheck **Send notifications for all branches**, and enter a comma-separated list of branches that you want to receive notifications" which we would have included if it was in `/help/generate-integration-url`. But, we can't add that to the help doc, since we can't selectively show that for the integrations with branch filtering available.

Reply:
> Add a new macro for the edit. Since all of these integrations except the GitHub doc use the exact same lines, we could add a new macro with that 1 line appended to the generate-url macro instruction.

Yes, this sounds like the right approach.